packages/gatsby/src/redux/actions/add-page-dependency.ts

```diff
@@ -0,0 +1,40 @@
+import { store } from "../"
+
+import { createPageDependency as internalCreatePageDependency } from "./internal"
+
+export const createPageDependency = ({
+  path,
+  nodeId,
+  connection,
+}: Parameters<typeof internalCreatePageDependency>[0]): void => {
```

Comment:
I would favor explicit types over this setup. Let's just change this to be a copy/paste of the types from the internal create pageDeps.